scripts/components/tool/tool-content.component.js

```diff
@@ -1,21 +1,27 @@
-
 export default class {
 
   onCreated() {
     this._setVars();
+
+    this.onSankeyReset = this._resetSankey.bind(this);
```

Comment:
It feels weird to bind in `onCreated` instead that in the `constructor`.

Reply:
We're using the same pattern in other places, I'm just reusing it